site/content/docs/5.0/getting-started/accessibility.md

```diff
@@ -47,6 +47,8 @@ For visually hidden interactive controls, such as traditional "skip" links, use
 
 Bootstrap includes support for the [`prefers-reduced-motion` media feature](https://drafts.csswg.org/mediaqueries-5/#prefers-reduced-motion). In browsers/environments that allow the user to specify their preference for reduced motion, most CSS transition effects in Bootstrap (for instance, when a modal dialog is opened or closed, or the sliding animation in carousels) will be disabled.
 
+Alternatively, the `scroll-behavior` property enables smooth scroll if the user did not ask for reduced motion.
```

Comment:
this, in isolation, makes no sense. alternatively to what? i think this needs a bit more of an explanation.

Reply:
Alternatively to disabling `transitions`s, in my mind :D 
Apart from the beginning of the sentence, using maybe something like "Bootstrap also use…" which would clarify the subject a bit, I'm not quite sure of what to explain more. I'll try something :)